blockchain/src/blockchain/mod.rs

```diff
@@ -432,6 +432,8 @@ impl<'env> Blockchain<'env> {
         }
 
         // Fork looks good.
+        // Drop read transaction.
+        read_txn.close();
```

Comment:
Weird, I get different line numbers in Github and IntelliJ. This is line 431 for me.